packages/clay-color-picker/src/Alpha.tsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {ClayInput} from '@clayui/form';
+import ClaySlider from '@clayui/slider';
+import React, {useState} from 'react';
+
+type Props = {
+	/**
+	 * Callback function for when the hue value changes
+	 */
+	onChange: (event: any) => void;
```

Comment:
The type for React event callback in this case can be: `(event: React.KeyboardEvent<HTMLInputElement>) => void`